src/components/BookForm/BookForm.css

```diff
@@ -0,0 +1,41 @@
+.form {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  width: 83vw;
+  margin: auto;
+  border-radius: 4px;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+}
+
+.input {
+  width: 70%;
+  height: 1.5rem;
+  padding: 15px;
+  border-radius: 4px;
+  font-size: large;
+  border: solid 1px #e8e8e8;
+  color: #c4c4c4;
+}
+
+.select {
+  width: 15rem;
+  padding: 10px;
+  height: 98%;
```

Comment:
- [ ] Please, let's make sure that styling is EXACTLY the same as the one provided in the designs. Your application has significant differences from the provided design. For example, the `Add New Book` section in the design is different in your application. As much as possible, try to use the font family and font sizes in the design. Please, pay attention to the margins ad padding as well. Please, see comparison between the two designs below:

Your Design
![Screenshot 2021-09-03 at 13-44-47 React App](https://user-images.githubusercontent.com/70150423/132015366-2b72e69b-a0f0-4a65-93f4-8c6d47b8b3f0.png)

Design Template
![Screenshot 2021-09-03 at 13-46-02 React Redux v 1 1](https://user-images.githubusercontent.com/70150423/132015574-22b0fdca-8133-47ea-85c0-785525ee6b70.png)

- [ ] Kindly use present tense for your commit messages. You have committed your code in past tenses. Commit messages are not what you have done, it's essentially what the code does. e.g `Style update button`.

Reply:
App Styling
![Screenshot from 2021-09-03 21-07-33](https://user-images.githubusercontent.com/73999611/132032097-a88ec405-51c2-4d88-9a30-47c433f570e8.png)

zeplin recommended
![Screenshot from 2021-09-03 21-06-59](https://user-images.githubusercontent.com/73999611/132032217-4ca00984-c768-488a-bede-17d84159e5fc.png)

Reply:
As per font-family is concerned it can't be implemented,
so far almost all the styles have been taken from the suggestion so that it resemble recommended site UI